src/Frontend/src/app/features/dashboard/dashboard.component.css

```
@@ -1,53 +1,191 @@
+}
+
+.dashboard-eyebrow {
+  font-size: 0.7rem;
+  font-weight: 700;
+  letter-spacing: 0.24em;
+  text-transform: uppercase;
+  color: hsl(var(--primary));
+}
+
+.dashboard-mono {
+  font-family: 'Geist Mono', 'Inter', monospace;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)